### ISSUE TITLE
Please decouple `linguify`

### ISSUE THREAD

**Issue by ivaquero** [jomaway/typst-gentle-clues#9]
The new release can't read my local `lang.toml`, I'm sure it's caused by `linguify:0.3.0` that force me to use the package's `lang.toml`.

<img width="373" alt="image" src="https://github.com/jomaway/typst-gentle-clues/assets/46377665/9a7d3234-0ff0-4cba-83f3-81d155c6a173">

Could you please decouple `linguify`? I am not sure if it's against your design, however, I think fewer dependencies makes this package more flexible.

**Comment by jomaway** [comment]
How did you load your `lang.toml`? Did you use `linguify_set_databse(toml(your_lang.toml))` ?

**Comment by ivaquero** [comment]
Yes, I tried both of the functions, but get the same error.

**Comment by ivaquero** [comment]
Sure.

```toml
default-lang = "zh"

[en]
font = "Arial"
chapter = "Chapter"
content = "Content"
definition = "Definition"
theorem = "Theorem"
lemma = "Lemma"
corollary = "Corollary"
algorithm = "Algorithm"
alert = "Alert"
tip = "Tip"
eq = " Eq."

[zh]
font = "Songti SC"
chapter = "章节"
content = "主要内容"
definition = "定义"
theorem = "定理"
lemma = "引理"
corollary = "推论"
algorithm = "算法"
alert = "注意"
tip = "提示"
eq = "  式 "
```

**Comment by ivaquero** [comment]
BTW, version 0.6.0 works fine.

**Comment by jomaway** [comment]
yes i know, linguify is a project which evolved out of gentle-clues. 
So i thought i extract it but i haven't thought about that it will affect if more than one package is using linguify. 
So there are two options. 
- One to use linguify only in templates
- Or make it possible to use in multiple packages and doesn't affect each other, which was the idea behind it. 
So it is actually a issue with the current version of linguify.

**Comment by ivaquero** [comment]
`linguify` is a great package, because of it, I started to follow you, and then found `gentle-clue`.

In my experience, I think the 1st option fits more cases. Also, I also agree the 2nd one is feasible if you make `linguify` a soft dependency.

**Comment by jomaway** [comment]
Thanks, i will try to resolve this a soon as possible. 
What you could try as a workaround is to copy the sections from gentle-clues lang.toml into your lang.toml if this works. 
Otherwise i guess it is the best to stay with version 0.6.0 until this is fixed.

**Comment by jomaway** [comment]
The issue should be resolved in the version 0.7.1

**Comment by ivaquero** [comment]
I tested, it works. Thank you for your efforts.
